standard/expressions.md

```diff
@@ -1447,20 +1451,25 @@ In a member access of the form `E.I`, if `E` is a single identifier, and if the
 > ```csharp
 > struct Color
 > {
->    public static readonly Color White = new Color(...);
->    public static readonly Color Black = new Color(...);
->    public Color Complement() {...}
+>     public static readonly Color White = new Color(...);
+>     public static readonly Color Black = new Color(...);
+>     public Color Complement() {...}
 > }
+>
 > class A
 > {
->    public **Color** Color; // Field Color of type Color
->    void F() {
->       Color = **Color**.Black; // Refs Color.Black static member
->       Color = Color.Complement(); // Invokes Complement() on Color fld
->    }
->    static void G() {
->       **Color** c = **Color**.White; // Refs Color.White static member
->    }
+>     public Color Color;              // Field Color of type Color
+>
+>     void F()
+>     {
+>         Color = Color.Black;         // Refers to Color.Black static member
+>         Color = Color.Complement();  // Invokes Complement() on Color fld
+>     }
+>
+>     static void G()
+>     {
+>         Color c = Color.White;       // Refers to Color.White static member
+>     }
```

Comment:
Filed <https://github.com/dotnet/csharpstandard/issues/541> about the loss of `**` delimiters that are still referenced by the paragraph below the code sample.